File: Wildcat All-In-One Timer/BreakPausedForm.cs
using System;
using System.Drawing;
using System.Globalization;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace UTimer
{
    public partial class BreakPausedForm
    {
        public BreakPausedForm()
        {
            InitializeComponent();
            _Button1.Name = "Button1";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            object sapi;
            sapi = Interaction.CreateObject("Sapi.spvoice");
            sapi.speak("Breaktime Resumed.");
            My.MyProject.Computer.Audio.Play(Application.StartupPath + @"\timesup4.wav");
            My.MyProject.Forms.BreakForm.Show();
            var myCulture = CultureInfo.CurrentCulture;
            var dayOfWeek = myCulture.Calendar.GetDayOfWeek(DateTime.Today);
            string dayName = myCulture.DateTimeFormat.GetDayName(dayOfWeek);
            var datetime = DateTime.Now;
            var datew = DateTime.Now;
            DateTime hourtime = Conversions.ToDate(datetime.Add(new TimeSpan(My.MyProject.Forms.BreakForm.hours, My.MyProject.Forms.BreakForm.minutes, My.MyProject.Forms.BreakForm.seconds)).ToString("HH:mm:ss tt", CultureInfo.InvariantCulture));
            var newdatetime = datew.Add(new TimeSpan(My.MyProject.Forms.BreakForm.hours, My.MyProject.Forms.BreakForm.minutes, My.MyProject.Forms.BreakForm.seconds));
            My.MyProject.Forms.BreakForm.Breaktime();
            My.MyProject.Forms.BreakForm.BreakTimer.Start();
            Close();
        }

        private void BreakDialog_Load(object sender, EventArgs e)
        {
            BackColor = ColorTranslator.FromHtml(My.MySettingsProperty.Settings.BackColor.ToArgb().ToString());
            ForeColor = ColorTranslator.FromHtml(My.MySettingsProperty.Settings.ForeColor.ToArgb().ToString());
        }
    }
}

File: Wildcat All-In-One Timer/BreakPausedForm.vb
Imports System.Globalization

Public Class BreakPausedForm
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim sapi As New SpeechLib.Synthesis.SpeechSynthesis

        sapi.Speak("Breaktime Resumed.")
        My.Computer.Audio.Play(Application.StartupPath & "\timesup4.wav")
        BreakForm.Show()
        Dim myCulture As System.Globalization.CultureInfo = Globalization.CultureInfo.CurrentCulture
        Dim dayOfWeek As DayOfWeek = myCulture.Calendar.GetDayOfWeek(Date.Today)
        Dim dayName As String = myCulture.DateTimeFormat.GetDayName(dayOfWeek)
        Dim datetime As DateTime = Date.Now
        Dim datew As DateTime = Date.Now
        Dim hourtime As DateTime = CDate(datetime.Add(New TimeSpan(BreakForm.hours, BreakForm.minutes, BreakForm.seconds)).ToString("HH:mm:ss tt", CultureInfo.InvariantCulture))
        Dim newdatetime As DateTime = datew.Add(New TimeSpan(BreakForm.hours, BreakForm.minutes, BreakForm.seconds))
        BreakForm.Breaktime()
        BreakForm.BreakTimer.Start()
        Me.Close()
    End Sub

    Private Sub BreakDialog_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Me.BackColor = System.Drawing.ColorTranslator.FromHtml(CStr(My.Settings.BackColor.ToArgb))
        Me.ForeColor = System.Drawing.ColorTranslator.FromHtml(CStr(My.Settings.ForeColor.ToArgb))
    End Sub
End Class